dvc/remote/base.py

```diff
@@ -80,6 +77,7 @@ class RemoteBASE(object):
 
     PARAM_RELPATH = "relpath"
     CHECKSUM_DIR_SUFFIX = ".dir"
+    CHECKSUM_JOBS = max(1, min(4, cpu_count() // 2))
```

Comment:
Is it right to use the local `cpu_count()` for the remote job count heuristics?

Reply:
This is not used for remotes, only for local

Reply:
@Suor Then maybe let's set it that way in RemoteLOCAL?

Reply:
Code location is wrong, but it's hard to move it without duplicating it. We should rethink all `jobs/checksum_jobs/--jobs` mess, created an issue for that #2295.

Anyway this one is not about it, so will just switch to conservative setting of 4.

Reply:
@efiop when this code:
https://github.com/iterative/dvc/blob/432373fb42c7b2430cc2fc20b70d3dee67a4aaa7/dvc/remote/base.py#L111-L114
will break for all remotes but local and ssh. Let's not mix it here I created a separate issue for that.